views/js/e2e/runner/_cleanup/cleanupData.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA ;
+ */
+
+ /**
+ * Data object
+ */
+export default {
+    testsPageUrl: 'tao/Main/index?structure=tests&ext=taoTests&section=manage_tests&uri=http%3A%2F%2Fwww.tao.lu%2FOntologies%2FTAOTest.rdf%23Test',
+    deliveriesPageUrl: 'tao/Main/index?structure=delivery&ext=taoDeliveryRdf&section=manage_delivery_assembly&uri=http%3A%2F%2Fwww.tao.lu%2FOntologies%2FTAODelivery.rdf%23AssembledDelivery'
```

Comment:
We need to think about if all the urls should be hardcoded in many different extensions, or if we construct them from pieces stored in a central location. For example, we already did that with `tao/Main/index` + the query params, all stored in `tao-e2e-runner`.

Reply:
Yes, I agree it is ugly a bit here but did not want to pollute the tao-e2e-runner with all the urls. Have no better idea atm. @btamas @krampstudio ?

Reply:
I prefer this solution, if these urls are related to this specific extension.